src/components/NavButton/NavButton.tsx

```diff
@@ -3,7 +3,7 @@ import {Link, Route} from 'react-router-dom';
 import './NavButton.css';
 
 interface NavButtonProps {
-    button: Button;
+    button: NavButton;
```

Comment:
I might be nitpicky, and feel free to dismiss this... we now have 2 "sibling" button classes, NavButton and ActionButton, that take in their props differently: in NavButton we group them into a type, in ActionButton we take them in separately. Do we want to uniformize the way button are created for a more coherent coding experience?